pycd01-sp2

chakay37 opened the pull request:
https://eso.vse.cz/~pycd01/sp2/

chakay37 committed:
init

chakay37 committed:
almost complete game

chybi win condition

chakay37 committed:
Delete www/pycd01/sp2/readme.md

chakay37 committed:
final commit - game done

chakay37 committed:
change of title tag

nvbach91 reviewed (changes requested):
- po skonceni hry umoznit zapis skore, nickname, avatar timestamp do localstorage
- umoznit spravovani vysledku her (mazani, serazeni, ...)
- umoznit vygenerovani custom avataru pomoci uzivatelskeho vstupu

chakay37 committed:
nickname system for scoreboard

chakay37 committed:
final odevzdání

náhodná výška míčku od druhého kola.
tabulka výsledků v local storage.
reset tlačítko tabulky a local storage

chakay37 commented:
náhodná výška míčku od druhého kola.
náhodný směr míčku.
tabulka výsledků v local storage.
reset tlačítko tabulky a local storage.
